### DISCUSSION TITLE
Export for web

### DISCUSSION THREAD

**Discussion by i2z1** [freeplane/freeplane#1068] [Help] [answered]
Hello everyone!
Is the best option (for pretty appearance) to use generated by Freeplane mindmap on website is the **export to XHTML (javascript)**?
Is there some ways to export to **JSON** for further usage with [Highcharts JS](https://www.highcharts.com/), for example as https://www.highcharts.com/demo/organization-chart ?

**Comment by EdoFro** [comment, marked as answer]
> Is there some ways to export to **JSON** for further usage with [Highcharts JS](https://www.highcharts.com/), for example as https://www.highcharts.com/demo/organization-chart ?

using groovy scripts

more info:
https://docs.freeplane.org/scripting/Scripting.html

https://groovy-lang.org/processing-json.html
